container/Home/Search/Search.style.js

```diff
@@ -108,7 +103,145 @@ export const SearchWrapper = styled.div`
     margin-bottom: 10px;
   }
 `;
+export const FormWrapperHome = styled.div`
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-row-gap: 8px;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: ${themeGet('color.2', '#F7F7F7')};
+  border-radius: 3px 0 0 3px;
+  /* min-height: 60px; */
+  margin-top: 10px;
+
+  .ant-checkbox-checked .ant-checkbox-inner {
+    background-color: ${themeGet('primary.0', '#008489')};
+    border-color: ${themeGet('primary.0', '#008489')};
+  }
+
+  form{
```

Comment:
The way is create a class to those html tag that you want edit

Reply:
ok i will change it

Reply:
I think that's right. They are using HTML tags inside of a styled component.